### PR TITLE
added introduction for an os-related blog

### PR DESCRIPTION
**Pull request by d-sonuga**
I added an introduction for my blog which is about writing a simple game from scratch in Rust.

### CODE REVIEW

File: content/this-month/2022-10/index.md [hunk 1]
```diff
@@ -64,6 +64,17 @@ In this section, we describe updates to Rust OS projects that are not directly r
     ...<<your project updates>>...
 -->
 
+### [`d-sonuga/blog-blasterball`](https://github.com/d-sonuga/blog-blasterball/)
+<span class="maintainers">(Section written by [@d-sonuga](https://github.com/d-sonuga))</span>
+
+<a href="https://d-sonuga.netlify.app" target="_blank">Demilade Sonuga's Blog</a> is a new blog on writing a
```
Comment (phil-opp) [inline content/this-month/2022-10/index.md:70]: You can just use a markdown link here:

```suggestion
[Demilade Sonuga's Blog](https://d-sonuga.netlify.app) is a new blog on writing a
```

File: content/this-month/2022-10/index.md [hunk 2]
```diff
@@ -64,6 +64,17 @@ In this section, we describe updates to Rust OS projects that are not directly r
...
+simple blasterball game with no OS, no engine and minimal external crates. The blog's aim is to take a Rust
+beginner from starting the project to finishing it and understanding everything involved in the project's
+creation.
+
+The project is still in it's early stages, so lots of feedback is very much needed.
+
+If you're interested, please take a look, and send your thoughts to @d_sonuga.
```
Comment (phil-opp) [inline content/this-month/2022-10/index.md:77]: You don't have a contact email on your GitHub profile, so I assume that you mean your twitter handle? In that case, we should be more specific and link it:

```suggestion
If you're interested, please take a look, and send your thoughts to [@d_sonuga](https://twitter.com/d_sonuga) on Twitter.
```

**Review by phil-opp — COMMENTED** [review]
Great project, thanks!

File: content/this-month/2022-10/index.md (hunk 2 above)
Comment (d-sonuga) [reply content/this-month/2022-10/index.md]: Thanks Phil.
I've made the changes.

**Review by phil-opp — APPROVED** [review]
Thanks!

**Pull request merged**